features/smells/manual_dispatch.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Smell - ManualDispatch
```

Comment:
We're trying to reduce the number of cucumber scenario's, and I don't think a separate scenario for an individual smell is very useful. So, as far as I'm concerned, this one can go. @troessner wdyt?